src/failed-spec-parser.js

```diff
@@ -1,13 +1,15 @@
-const FAILED_LINE = /at \[object Object\]\.<anonymous> \((.*)\)/g;
+const FAILED_LINES = /at \[object Object\]\.<anonymous> \((.*?):.*\)/g;
 
 export default function (output = '') {
-  // this could all probably fit into one regex...
-  var failedSpecLines = output.match(FAILED_LINE);
+  var output = arguments.length <= 0 || arguments[0] === undefined ? '' : arguments[0];
```

Comment:
This was actually copy&pasted from the code that npm install protractor-flake delivered. Otherwise I agree that it's not necessary.